nanoFramework.Device.Can/packages.config

```diff
@@ -1,6 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="nanoFramework.CoreLibrary" version="1.12.0" targetFramework="netnanoframework10" />
-  <package id="nanoFramework.Runtime.Events" version="1.10.0" targetFramework="netnanoframework10" />
-  <package id="Nerdbank.GitVersioning" version="3.1.91" developmentDependency="true" targetFramework="netnanoframework10" />
+  <package id="nanoFramework.CoreLibrary" version="1.12.0" targetFramework="netnano1.0" />
```

Comment:
missing targetFramework

Reply:
Should not need it... this is a dev dependency! (unless `nf,runtime.events` is also wrong?!)